N-Queens.py

```diff
@@ -0,0 +1,36 @@
+# Can you let me know what wronf I am doing? I am not able to understand.
+
```

Comment:
could you explain the approach you have used here, As per the code, I see you have missed some conditional statements in the helper function